templates/ntp.conf.erb

```diff
@@ -13,15 +15,17 @@ restrict -6 ::1
 <% if @servers.empty? -%>
 server  127.127.1.0 # local clock
 fudge 127.127.1.0 stratum <%= @stratum %>
+
 <% elsif !@virtual.eql?('vmware') -%>
 server  127.127.1.0 # local clock
 fudge 127.127.1.0 stratum 10
-<% end -%>
 
+<% end -%>
 <%
   t_servers = []
   if @servers.kind_of?(Array)
-    t_servers = @servers.map{|x| "server #{x}"}
+    options = @default_options.empty? ? '' : " #{Array(@default_options).join(' ')}"
```

Comment:
This can be replaced with just `Array(@default_options).join(' ')`. It returns `''` by default anyway.

Reply:
@lnemsick-simp Sorry, missed the beginning space at the second option.